### DISCUSSION TITLE
How can I build in autogen-studio from source code.

### DISCUSSION THREAD

**Discussion by pyanagiya** [microsoft/autogen#3418] [Q&A] [answered]
I read autogen-studio [README here](https://github.com/microsoft/autogen/tree/autogenstudio/samples/apps/autogen-studio)
and tried to install and build from source with `yarn build` command in frontend directory.
But, I got these errors stated below. 

```
ERROR #11321  PLUGIN

"gatsby-plugin-manifest" threw an error while running the onPostBootstrap lifecycle:

Input file contains unsupported image format



  Error: Input file contains unsupported image format

not finished onPostBootstrap - 0.036s
```

**Comment by tjroamer** [comment]
Got same error with latest version, does anyone has a solution for this?

**Reply by jackgerrits** [reply, marked as answer]
Make sure that git-lfs is setup and the files properly hydrated. It looks like the image pointers werent downloaded

**Reply by tjroamer** [reply]
Many thanks for the quick reply. Yes, I can confirm that git-lfs solved this issue. Just for others: what I did:

```
apt-get install git-lfs
git lfs fetch --all
git lfs checkout # this command downloads all missing image files to the working dir
```
After having done the above, the command `yarn build` ran through.

**Reply by mj-moor** [reply]
You mean you did this in the running container?
